tests/acceptance/institution-page/claim-offer-test.js

```diff
@@ -64,5 +64,81 @@ module('Acceptance | institution-page | claim-offer-test', function (hooks) {
     assert.ok(applicationModal.verifyEmailStepContainer.isVisible, 'Verify email step should be visible');
   });
 
+  test('can redo the flow from change email button', async function (assert) {
+    testScenario(this.server);
+    const institution = createInstitution(this.server, 'nus');
+    const user = signIn(this.owner, this.server);
+
+    this.server.create('institution-membership-grant-application', {
+      institution: institution,
+      user: user,
+      status: 'awaiting_verification',
+      normalizedEmailAddress: 'bill@u.nus.edu',
+      originalEmailAddress: 'bill@u.nus.edu',
+    });
+
+    await institutionPage.visit({ institution_slug: 'nus' });
+    await institutionPage.claimOfferButtons[0].click();
+    await animationsSettled();
+
+    const applicationModal = institutionPage.campusProgramApplicationModal;
+    assert.ok(applicationModal.isVisible);
+    assert.notOk(applicationModal.enterEmailStepContainer.isVisible, 'Enter email step should not be visible');
+    assert.ok(applicationModal.verifyEmailStepContainer.isVisible, 'Verify email step should be visible');
+    assert.strictEqual(applicationModal.verifyEmailStepContainer.emailAddress, 'bill@u.nus.edu');
+
+    await applicationModal.verifyEmailStepContainer.clickOnChangeEmailButton();
+
+    assert.ok(applicationModal.enterEmailStepContainer.isVisible, 'Enter email step should be visible');
+    assert.notOk(applicationModal.verifyEmailStepContainer.isVisible, 'Verify email step should not be visible');
+    assert.strictEqual(applicationModal.enterEmailStepContainer.emailAddressInputValue, 'bill@u.nus.edu');
+    await animationsSettled();
```

Comment:
This `animationsSettled()` call in particular seems frivolous - it happens **after** the assertions and the next action acts on the same thing that was present in the previous assertions. 

Review this and the other usages too, see if we really need them